sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/SearchIndexingBufferedSenderOptions.java

```diff
@@ -22,30 +25,49 @@
 public final class SearchIndexingBufferedSenderOptions<T> {
     private static final boolean DEFAULT_AUTO_FLUSH = true;
     private static final int DEFAULT_INITIAL_BATCH_ACTION_COUNT = 512;
-    private static final Duration DEFAULT_FLUSH_WINDOW = Duration.ofSeconds(60);
-    private static final int DEFAULT_MAX_RETRIES = 3;
-    private static final Duration DEFAULT_RETRY_DELAY = Duration.ofMillis(800);
-    private static final Duration DEFAULT_MAX_RETRY_DELAY = Duration.ofMinutes(1);
+//    private static final Function<Integer, Integer> DEFAULT_SCALE_DOWN_FUNCTION = oldBatchCount -> {
```

Comment:
Should the commented code be removed?

Reply:
Keeping the commented out code for now as this functionality may be added as options in the future